ieeeuoft/newhacks-site-2022
# removed rsvp

## Overview

- Resolves #NH-14
- removed rsvp


## Unit Tests Created

- removed rsvp unit tests


## Steps to QA

- apply as a hacker, fill out the application form
- check your dashboard for any indication of requiring to RSVP, there should be none
